mixin/main.go

```diff
@@ -0,0 +1,313 @@
+package main
+
+import (
+	"crypto/rsa"
+	"crypto/x509"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"regexp"
+	"runtime"
+	"strconv"
+
+	"github.com/clearlinux/mixer-tools/builder"
+	"github.com/clearlinux/mixer-tools/helpers"
+	"github.com/clearlinux/mixer-tools/swupd"
+	"github.com/pkg/errors"
+)
+
+var mixWS = "/usr/share/mix"
+var config = "/usr/share/mix/builder.conf"
+
+const builderConf = `[Mixer]
+LOCAL_BUNDLE_DIR = /usr/share/mix/local-bundles
+
+[Builder]
+SERVER_STATE_DIR = /usr/share/mix/update
+BUNDLE_DIR = /usr/share/mix/local-bundles
+YUM_CONF = /usr/share/mix/.yum-mix.conf
+CERT = /usr/share/mix/Swupd_Root.pem
+VERSIONS_PATH =/usr/share/mix
+LOCAL_RPM_DIR = /usr/share/mix/local-rpms
+LOCAL_REPO_DIR = /usr/share/mix/local
+
+[swupd]
+BUNDLE=os-core
+CONTENTURL=file:///usr/share/mix/update/www
+VERSIONURL=file:///usr/share/mix/update/www
+FORMAT=1
```

Comment:
I'm wondering if you could reuse MixConfig here instead, if now now, then when it moves to its own package. Maybe I could change LoadDefaults() to accept a patch instead of always using pwd. This would also remove the need for setUpMixDir()